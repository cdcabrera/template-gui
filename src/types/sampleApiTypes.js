const SAMPLE_API_RESPONSE_DATA = 'data';
const SAMPLE_API_RESPONSE_DATA_DATE = 'date';
const SAMPLE_API_RESPONSE_DATA_LOREM = 'lorem';
const SAMPLE_API_RESPONSE_DATA_IPSUM = 'ipsum';

const sampleApiTypes = {
  SAMPLE_API_RESPONSE_DATA,
  SAMPLE_API_RESPONSE_DATA_DATE,
  SAMPLE_API_RESPONSE_DATA_LOREM,
  SAMPLE_API_RESPONSE_DATA_IPSUM
};

export {
  sampleApiTypes as default,
  sampleApiTypes,
  SAMPLE_API_RESPONSE_DATA,
  SAMPLE_API_RESPONSE_DATA_DATE,
  SAMPLE_API_RESPONSE_DATA_LOREM,
  SAMPLE_API_RESPONSE_DATA_IPSUM
};
