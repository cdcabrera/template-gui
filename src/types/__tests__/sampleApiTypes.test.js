import sampleApiTypes, * as allApiTypes from '../sampleApiTypes';

describe('SampleApiTypes', () => {
  it('Should have specific API properties', () => {
    expect(sampleApiTypes).toMatchSnapshot('specific types');
    expect(allApiTypes).toMatchSnapshot('all exported api types');
  });
});
