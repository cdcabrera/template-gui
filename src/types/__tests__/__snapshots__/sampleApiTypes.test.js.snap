// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleApiTypes Should have specific API properties: all exported api types 1`] = `
Object {
  "SAMPLE_API_RESPONSE_DATA": "data",
  "SAMPLE_API_RESPONSE_DATA_DATE": "date",
  "SAMPLE_API_RESPONSE_DATA_IPSUM": "ipsum",
  "SAMPLE_API_RESPONSE_DATA_LOREM": "lorem",
  "default": Object {
    "SAMPLE_API_RESPONSE_DATA": "data",
    "SAMPLE_API_RESPONSE_DATA_DATE": "date",
    "SAMPLE_API_RESPONSE_DATA_IPSUM": "ipsum",
    "SAMPLE_API_RESPONSE_DATA_LOREM": "lorem",
  },
  "sampleApiTypes": Object {
    "SAMPLE_API_RESPONSE_DATA": "data",
    "SAMPLE_API_RESPONSE_DATA_DATE": "date",
    "SAMPLE_API_RESPONSE_DATA_IPSUM": "ipsum",
    "SAMPLE_API_RESPONSE_DATA_LOREM": "lorem",
  },
}
`;

exports[`SampleApiTypes Should have specific API properties: specific types 1`] = `
Object {
  "SAMPLE_API_RESPONSE_DATA": "data",
  "SAMPLE_API_RESPONSE_DATA_DATE": "date",
  "SAMPLE_API_RESPONSE_DATA_IPSUM": "ipsum",
  "SAMPLE_API_RESPONSE_DATA_LOREM": "lorem",
}
`;
